src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -738,11 +825,20 @@ public void ClusterSharding_should_be_easy_to_use_with_extensions()
                     //#counter-start
                     ClusterSharding.Get(Sys).Start(
                         typeName: "AnotherCounter",
-                        entityProps: Props.Create<Counter>(),
+                        entityProps: Props.Create<AnotherCounter>(),
                         settings: ClusterShardingSettings.Create(Sys),
                         idExtractor: Counter.ExtractEntityId,
                         shardResolver: Counter.ExtractShardId);
-                }, _third, _fourth, _fifth, _sixth);
+
+                    //#counter-supervisor-start
+                    ClusterSharding.Get(Sys).Start(
+                      typeName: "SupervisedCounter",
+                      entityProps: Props.Create<CounterSupervisor>(),
+                      settings: ClusterShardingSettings.Create(Sys),
+                      idExtractor: Counter.ExtractEntityId,
+                      shardResolver: Counter.ExtractShardId);
```

Comment:
Originally I have rename idExtractor & shardResolver to match akka, but it increased number of changes quite a lot. So I've reverted it to keep the # of changes minimal. If you want, I can rename them